src/views/home/index.js

```diff
@@ -2,17 +2,20 @@ import React from 'react';
 import Footer from '../../components/footer';
 import Header from '../../components/header';
 import Menu from '../../components/menu';
+import Gallery from '../../components/gallery';
 import './home.css';
 
-function Home() {
+function Home({sessions}) {
+    console.log("Printing session..\n")
     return (
         <div className="Home">
             {/* TODO: HOME VIEW */}
             <Header/>
             <Menu/>
-            <div>
-                TODO: GALLERY COMPONENT
+            <div className="gallery__container">
```

Comment:
Maybe gallery__container should be the Gallery as it's own component and your current Gallery component should be a Session component, so like:
- Gallery
	- Session
		- Snapshot, Snapshot, Snapshot...
	- Session
		- Snapshot, Snapshot, Snapshot...
...